package.json

```diff
@@ -5175,6 +5175,168 @@
           "markupCommentPunctuation": [
             "punctuation.definition.comment.html",
             "comment.block.html"
+          ],
+          "keyword": [
```

Comment:
This is needed to override the default colors VS Code uses since they can differ compared to what C# uses. For example, `keyword` is colored purple by VS Code, but colored blue by C#.

Thanks @dibarbet for the tip!

Reply:
Interesting, so "keyword.cs" is already blue for VS Code and it just... knows?

Reply:
I'm guessing C# sets the color somewhere, although I'm not sure where. Maybe @dibarbet or @JoeRobich would know?

Reply:
I don't quite remember - it was potentially that the lsp token type for `keyword` mapped to `keyword.control` but `keyword.cs` falls back to `keyword` instead

Reply:
POssibly https://github.com/microsoft/vscode/blob/b96b558f6479557a43de966d7cfdbcdffb82d715/src/vs/platform/theme/common/tokenClassificationRegistry.ts#L545